cmd/adwatchd/commands/run.go

```diff
@@ -18,12 +18,8 @@ func (a *App) installRun() {
 	cmd := &cobra.Command{
 		Use:   "run",
 		Short: i18n.G("Starts the directory watch loop"),
-		Long: i18n.G(`Can run as a service through the service manager or interactively as a standalone application.
-
-The program will monitor the configured directories for changes and bump the appropriate GPT.ini versions anytime a change is detected.
-If a GPT.ini file does not exist for a directory, a warning will be issued and the file will be created. If the GPT.ini file is incompatible or malformed, the program will report an error.
-`),
-		Args: cobra.NoArgs,
+		Long:  i18n.G("Can run as a service through the service manager or interactively as a standalone application.\n\nThe program will monitor the configured directories for changes and bump the appropriate GPT.ini versions anytime a change is detected.\nIf a GPT.ini file does not exist for a directory, a warning will be issued and the file will be created. If the GPT.ini file is incompatible or malformed, the program will report an error.\n"),
```

Comment:
```suggestion
Long:  i18n.G("Can run as a service through the service manager or interactively as a standalone application.\n\nThe program will monitor the configured directories for changes and bump the appropriate GPT.ini versions anytime a change is detected.\nIf a GPT.ini file does not exist for a directory, a warning will be issued and the file will be created.\nIf the GPT.ini file is incompatible or malformed, the program will report an error.\n"),
```

The `If the GPT.init...` sentence lives in its own line.

Reply:
Viewing the diff in the "files" tab made me think that there is:
![image](https://user-images.githubusercontent.com/11138291/228859351-8d96b65b-429c-4a7a-ae5c-48d6706776a5.png)